### DISCUSSION TITLE
graphql-request change operation names to camelCase

### DISCUSSION THREAD

**Discussion by lanwin** [dotansimha/graphql-code-generator#9119] [Q&A]
Hi, I am looking for a way to apply a naming convention only to the generated option names. 

Example:
```
query FetchUsers {...}

getSDK().FetchUsers() should be getSDK().fetchUsers()
```

From what I can see it seems I can only apply naming conventions to the generated types and enums but not on the generated operation names.

Is there a way I overlook?

**Comment by saihaj** [comment]
Can you share a [minimal reproduction](https://en.wikipedia.org/wiki/Minimal_working_example). You can either use [StackBlitz](https://stackblitz.com/), [CodeSandbox](https://codesandbox.io/) or a public GitHub repo.
